blog-posts/the-modern-code-reviewer-2021.md

```diff
@@ -0,0 +1,219 @@
+
+There is a total of three code review approaches to help your team improving their reviewing throughput. We introduce the **shallow review**, the **common review** and the **expert review** which primarily differ in the reviewing intensity. The more intense the higher the amount of time you have to invest. Your average code review probably corresponds to the common review with some reviews being of the expert format.
+
+The novelty of the approach is the explicitly given choice. A contribution guideline rarely describes how to conduct a review so people usually conduct code reviews following their gut feeling and to fit expectations. This means it highly depends on the person how a review is conducted. You are the alpha geek of your team? Of course you have to go into depth for every single PR which will make you a bottle neck if everyone relies on your reviews. Your are a less experienced developer? You will probably conduct a common review which takes high amounts of time for you, because of the overwhelming complexity or size of your project. You are new to the team and you fear to provide a common review so you decide not to take part in the review process at all.
+
+This code review pattern gives you a choice so you can fit more reviews in the same amount of time while dedicating enough time to the important changes while reducing your investment on trivial changes or things that can easily be changed if not perfect.
+
+## The shallow review
+
+T**he shallow review** is best ****suited for peers who are in a hurry (lacking time), not deeply involved (lacking knowledge) or feel intimidated by the code or experience of the reviewee (lacking confidence). A majority of pull requests should be suitable for this code review type, as long and they have a small enough size. When a code change introduces major architectural changes, people with shallow reviews are still welcomed but the reviewee must additionally require in-depth reviews.
+
+### The reviewer's cost
+
+Typically such a shallow review should be completed within 5 to 10 minutes. That will allow you to conduct multiple pull request reviews per day. **The cognitive load is rather low**, so you can squeeze them in whenever you transition to another task or before or after breaks. The review type is ideal for small features and additions, fixes and code refactorings.
+
+### How to conduct
+
+- Carefully read the PR description or any linked ticket to understand the problem
+- Glimpse through the PR to check in your mind if that indeed could solve the problem or deliver the feature.
+- Leave some positive comments or any doubts.
+- Feel free to ask a question for something you don't understand
+- Phrase some summary and submit. done. feasible in some cases even in less than 5 minutes.
```

Comment:
```suggestion
- Phrase some summary and submit. Done. Feasible in some cases even in less than 5 minutes.
```

Reply:
I have rephrased it to sound less colloquial🙏
> Phrase some summary and then submit to complete the review.